app/controllers/permissions_controller.rb

```diff
@@ -43,9 +43,14 @@ def render_invalid_params_json
            status: :unprocessable_entity
   end
 
-  def render_permission_error_json(permission)
-    render json: { errors: permission.errors.full_messages },
-           status: :unprocessable_entity
+  def render_unsuccessful_permission_json(permission)
+    errors = permission.errors.full_messages
+    if errors.include?('User token is already associated with school urn')
```

Comment:
Can we make this condition more specific. Something like the attached suggestion
```suggestion
 if permission.errors.added?(:user_token, :taken)
```